src/utils/growthbook.ts

```diff
@@ -0,0 +1,20 @@
+import { GrowthBook } from "@growthbook/growthbook-react"
+
+const GROWTHBOOK_API_HOST = "https://cdn.growthbook.io"
+
+export const getGrowthBookInstance = (clientKey: string, isDev = false) => {
+  return new GrowthBook({
+    apiHost: GROWTHBOOK_API_HOST,
+    clientKey,
+    enableDevMode: isDev, // enable only for local dev
```

Comment:
this is never set to `true` - wdyt about removing this argument from `getGrowthBookInstance` and setting it based on whether `REACT_APP_ENV === LOCAL_DEV`

Reply:
I thought about that but in our local dev, our react env is not LOCAL_DEV as per 1PW

Also the intended use is that if devs need this, they add the `true` to the constructor in App.jsx and by default it will remain false

Reply:
hmm, my main worry is that this value should be true by default. this will result in q abit of overhead when switching between branches, for example (have to stash, switch pop etc), which is why i thought it might make sense to dynamically compute this on a per env basis. 

the tradeoff we're making here seems to be the time taken to standardise `REACT_APP_ENV` versus how often this might be swapped to `true`. 

if it makes sense for this to be **false by default**, feel free to resolve this comment. otherwise, maybe just put a message out in the slack channel to standardise this would be ok toO!

Reply:
oh cos that's the production flag - so imagine if you always want to see the new dev stuff, you'd always need to stash then pop

Reply:
more inclined to do what you've laid out above - with the only caveat being that we align `localDev` cos got (non dev) envs that we might wanna have - eg: `uat/vapt`

Reply:
Yea exactly, so we need to be firm on the definition and there could be future cases where we introduce new environments. To be on the safe side, the flag needs to be a whitelist (assert if specific environment exists) rather than asserting some environment does not match

Reply:
can, sounds good! i'll approve it first - feel free to merge once `isDev` flag is in

Reply:
added